server/routes/debug.ts

```diff
@@ -76,35 +142,87 @@ export default function makeDebugRoutes(services: Services): Record<string, Requ
       if (toDateInput) {
         queryString.append('toDate', toDateInput)
       }
+      if (incidentType) {
+        queryString.append('incidentType', incidentType)
+      }
+      if (incidentStatuses) {
+        queryString.append('incidentStatuses', incidentStatuses)
+      }
+      const tableHeadUrlPrefix = `/incidents?${queryString}&`
+      if (sort) {
+        queryString.append('sort', sort)
+      }
+      if (order) {
+        queryString.append('order', order)
+      }
 
       const urlPrefix = `/incidents?${queryString}&`
-      const paginationParams = pagination(
-        pageNumber,
-        eventsResponse.totalPages,
-        urlPrefix,
-        'moj',
-        eventsResponse.totalElements,
-        eventsResponse.size,
-      )
-      const noFiltersSupplied = Boolean(!prisonId && !fromDate && !toDate)
 
-      const events = eventsResponse.content
-      const usernames = events.map(event => event.modifiedBy)
+      const noFiltersSupplied = Boolean(!prisonId && !fromDate && !toDate && !incidentType && !incidentStatuses)
```

Comment:
should there be `reportingOfficer` in here too?

Reply:
It will be, I just hadn't put it in yet as it's not an active filter as such yet, as it doesn't yet do anything when you select it.